dipy/segment/mask.py

```diff
@@ -102,7 +102,7 @@ def bounding_box(vol):
 
 
 def crop(vol, mins, maxs):
-    """ Crops the input volume.
+    """Crops the input volume.
```

Comment:
I set up my editor to enforce PEP257 (https://pypi.python.org/pypi/pep257), and it flagged that.